2.cellprofiler_processing/cputils.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+from pathlib import Path
+
+def run_cellprofiler(path_to_pipeline: str, path_to_output: str, path_to_images: str):
+    """Profile batch with CellProfiler (runs segmentation and feature extraction)
+
+    Args:
+        path_to_pipeline (str): path to the CellProfiler .cppipe file with the segmentation and feature measurement modules
+        path_to_output (str): path to the output folder for the .sqlite file
+        path_to_images (str): path to folder with IC images from specific plate
+    """
+    # currently, the output path in the command does not output correctly, so the directory of the terminal must be set to the output path
+    os.chdir(path_to_output)
```

Comment:
nice quick fix. I imagine this is what Beth's comment might be able to help with?

Reply:
@gwaybio 

Yup! She was able to help and I have gotten rid of this part of the code. 

You will see in the next commit the solution.